59397: Update entry for Domain Account Password in directoryservices.rst (#595)
freenas/freenas-docs: userguide/intro.rst

* The :guilabel:`MSDOSFS locale` drop-down menu has been added to
  :menuselection:`Storage --> Import Disk`.

> * The :guilabel:`Domain Account Password` field in

Reviewer: ```suggestion
* A :guilabel:`Domain Account Password` in
```

  :menuselection:`Directory Services --> Active Directory`

>   has been reworked to only be required when configuring a domain for

Reviewer: ```suggestion
  is only required when configuring a domain for
```
